commons/pom.xml

```diff
@@ -38,6 +46,19 @@
         <groupId>net.revelc.code.formatter</groupId>
         <artifactId>formatter-maven-plugin</artifactId>
       </plugin>
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-source-plugin</artifactId>
+        <executions>
+          <execution>
+            <id>attach-sources</id>
+            <phase>verify</phase>
+            <goals>
+              <goal>jar-no-fork</goal>
+            </goals>
+          </execution>
+        </executions>
+      </plugin>
```

Comment:
:+1: for generating the sources jar (even if the idea decompiler plugin is so nice than I generally don't have the need for them, or the "get the sources from the Internet" feature from idea works pretty well also).

Any motivation to add them from your side ?

Reply:
well, eclipse doesn't seem to be that smart